.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: python
 python:
-  - "3.5"
   - "3.6"
   - "3.7"
-  - "pypy3.5"
```

Comment:
I believe there's a PyPy 3.6 we should have here.

Reply:
travis-ci said that it fail at pypy3.6